src/utility.ts

```diff
@@ -286,6 +276,19 @@ export class Utility {
         }
     }
 
+    public static async getInputDeviceList(eventName: string, onlyEdgeDevice: boolean = false, iotHubConnectionString?: string): Promise<DeviceItem[]> {

```

Comment:
what's the purpose of extracting this function?

Reply:
Resolved.